tests/test_gcloud.py

```diff
@@ -10,6 +11,7 @@
 from google.cloud.storage.blob import Blob
 
 from storages.backends import gcloud
+from tests.test_s3boto3 import NonSeekableContentFile
```

Comment:
Can you create a `tests/util.py` and import from that. I don't want to cross-import the various backend's test files into each other.